### ISSUE TITLE
Would like to know how to implement it into own project i

### ISSUE THREAD

**Issue by nsharan2000** [shroominic/codeinterpreter-api#63]
the implementation details are not so clear. Need a better documentation on how to implement it in a langchain project. I am currently using superagi, would like to see how it can be integrated with other tools as well

**Comment by shroominic** [comment]
yeah true we need better docs but have not found the time to do this yet
